src/components/templates/PosterHall/PosterHall.tsx

```diff
@@ -55,7 +56,7 @@ export const PosterHall: React.FC<PosterHallProps> = ({ venue }) => {
       </div>
 
       <div className="PosterHall__more-button">
-        {isPostersLoaded && (
+        {hasHiddenPosters && isPostersLoaded && (
```

Comment:
[NB] We should extract this into something like, then use that here:

```typescript
const shouldShowMorePosters = isPostersLoaded && hasHiddenPosters
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1460/commits/8051f616d516d57fde53964d6b45ad92e447b09f